### ISSUE TITLE
Docs to run with @iov/cli don't work out of the box

### ISSUE THREAD

**Issue by ethanfrey** [iov-one/iov-core#328]
Roman tried to install from npm and that didn't work, he had to install a few more packages manually.

Let's reproduce this on a clean vm (not dev machine) and verify what commands are needed and update documentation to make this process smoother for those not cloning the repo and building it.

Please assign to Ethan when there is a clear way to reproduce problem.

**Comment by webmaster128** [comment]
Let me take over this one (reproduce and fix) as lehajam asked me to support him with a similar or equal issue

**Comment by ruseinov** [comment]
@webmaster128 hey Simon, sure.
I think the issue was that I installed iov-cli from npm, but didn't yarn install in this repo, which led to cli not loading some libs.

I don't remember exact steps to reproduce, but you can start there, making sure you have a clean env that does not have these deps already installed.

There were some errors about typescript not compiling when running cli, but I assume it was happening because it was missing deps.

**Comment by webmaster128** [comment]
When installing @iov/cli globally (which I think was never tested during development), the iov-cli binary is installed at `/usr/local/lib/node_modules/@iov/cli/bin/iov-cli` and `__dirname` is `/usr/local/lib/node_modules/@iov/cli/bin`. All the dependencies are in `/usr/local/lib/node_modules/@iov/cli/node_modules`.

Now

```
$ cd /usr/local/lib/node_modules/@iov/cli && node -p "module.paths"
[ '/usr/local/lib/node_modules/@iov/cli/node_modules',
  '/usr/local/lib/node_modules/@iov/node_modules',
  '/usr/local/lib/node_modules',
  '/usr/local/node_modules',
  '/usr/node_modules',
  '/node_modules' ]
```

includes `/usr/local/lib/node_modules/@iov/cli/node_modules`. However,

```
$ cd ~ && iov-cli
>> module.paths
[ '/home/simon/repl/node_modules',
  '/home/simon/node_modules',
  '/home/node_modules',
  '/node_modules',
  '/home/simon/.node_modules',
  '/home/simon/.node_libraries',
  '/usr/lib/nodejs' ]
```

does not include `/usr/local/lib/node_modules/@iov/cli/node_modules`. So one would need to inject the correct path into module.paths for both JS execution and TypeScript compiler.

**Comment by webmaster128** [comment]
Setting `baseUrl` for compile time and `module.paths` for run time module resolution should do the trick (https://github.com/iov-one/iov-core/pull/340). Will do further testing tomorrow

**Comment by webmaster128** [comment]
I can reproduce the issue in the CI: https://github.com/iov-one/iov-core/pull/341 However, the fix from #340 is not yet working
